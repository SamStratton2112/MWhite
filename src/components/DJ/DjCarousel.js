import { Carousel, Container } from 'react-bootstrap';
import Video from './Video';
import mike from '../../data';

const DjCarousel = () => {
    return (
        <Container>
            <Carousel className='align-items-center'>
                <Carousel.Item interval={5000}><Video id={mike.dj.links.video.liveARoll} /></Carousel.Item>
                <Carousel.Item interval={5000}><Video id={mike.dj.links.video.SirensCrush} /></Carousel.Item>
                <Carousel.Item interval={5000}><Video id={mike.dj.links.video.carnitas} /></Carousel.Item>
                <Carousel.Item interval={5000}><Video id={mike.dj.links.video.spiritsFest} /></Carousel.Item>
            </Carousel>
        </Container>
    );
};

export default DjCarousel;