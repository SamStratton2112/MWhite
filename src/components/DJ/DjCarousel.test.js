import DjCarousel from './DjCarousel';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

describe(DjCarousel, () => {
    it("renders without crashing", () => {
        render(<MemoryRouter>
            <DjCarousel />
        </MemoryRouter>);
    });
    it("matches snapshot", () => {
        const { asFragment } = render(<MemoryRouter>
            <DjCarousel />
        </MemoryRouter>);
        expect(asFragment()).toMatchSnapshot();
    });
});