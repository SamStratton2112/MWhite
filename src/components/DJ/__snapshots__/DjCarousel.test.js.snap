// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DjCarousel matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
    style="margin-top: 4em; padding-bottom: 10em;"
  >
    <div
      class="align-items-center carousel slide"
    >
      <div
        class="carousel-indicators"
      >
        <button
          aria-current="true"
          aria-label="Slide 1"
          class="active"
          data-bs-target=""
          type="button"
        />
        <button
          aria-current="false"
          aria-label="Slide 2"
          data-bs-target=""
          type="button"
        />
        <button
          aria-current="false"
          aria-label="Slide 3"
          data-bs-target=""
          type="button"
        />
        <button
          aria-current="false"
          aria-label="Slide 4"
          data-bs-target=""
          type="button"
        />
        <button
          aria-current="false"
          aria-label="Slide 5"
          data-bs-target=""
          type="button"
        />
      </div>
      <div
        class="carousel-inner"
      >
        <div
          class="active carousel-item"
          interval="5000"
        >
          <div
            class="container"
            style="width: 50%;"
          >
            <div
              class="ratio ratio-16x9"
            >
              <iframe
                allowfullscreen=""
                src="https://www.youtube.com/embed/BKKLGFTLobs?rel=0"
                title="YouTube video"
              />
            </div>
          </div>
        </div>
        <div
          class="carousel-item"
          interval="5000"
        >
          <div
            class="container"
            style="width: 50%;"
          >
            <div
              class="ratio ratio-16x9"
            >
              <iframe
                allowfullscreen=""
                src="https://www.youtube.com/embed/drABef-QGhg?rel=0"
                title="YouTube video"
              />
            </div>
          </div>
        </div>
        <div
          class="carousel-item"
          interval="5000"
        >
          <div
            class="container"
            style="width: 50%;"
          >
            <div
              class="ratio ratio-16x9"
            >
              <iframe
                allowfullscreen=""
                src="https://www.youtube.com/embed/vz0OvRqfbro?rel=0"
                title="YouTube video"
              />
            </div>
          </div>
        </div>
        <div
          class="carousel-item"
          interval="5000"
        >
          <div
            class="container"
            style="width: 50%;"
          >
            <div
              class="ratio ratio-16x9"
            >
              <iframe
                allowfullscreen=""
                src="https://www.youtube.com/embed/MNHSMyaNZ6Y?rel=0"
                title="YouTube video"
              />
            </div>
          </div>
        </div>
        <div
          class="carousel-item"
          interval="5000"
        >
          <div
            class="container"
            style="width: 50%;"
          >
            <div
              class="ratio ratio-16x9"
            >
              <iframe
                allowfullscreen=""
                src="https://www.youtube.com/embed/5ZcIKeqHBuo?rel=0"
                title="YouTube video"
              />
            </div>
          </div>
        </div>
      </div>
      <a
        class="carousel-control-prev"
        href="#"
        role="button"
        tabindex="0"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
        <span
          class="visually-hidden"
        >
          Previous
        </span>
      </a>
      <a
        class="carousel-control-next"
        href="#"
        role="button"
        tabindex="0"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
        <span
          class="visually-hidden"
        >
          Next
        </span>
      </a>
    </div>
  </div>
</DocumentFragment>
`;
